src/LandIce/evaluators/LandIce_EnthalpyResid.hpp

```diff
@@ -91,7 +91,38 @@ class EnthalpyResid : public PHX::EvaluatorWithBaseImpl<Traits>,
   double flux_reg_alpha;
   double flux_reg_beta;
 
+  const double powm3 = 1e-3;  //[k], k=1000
+  const double powm6 = 1e-6;  //[k^2], k=1000
+  const double pow3 = 1e3;  //[k^{-1}], k=1000
+
```

Comment:
I'd go ahead and make these `static constexpr` instead of `const` and hopefully the compiler will optimize these out. Replace `const` with `constexpr` in the other kernels too whenever the value is known at compile time.

Reply:
If you make them constexp, you might just keep them in the `_Def.hpp` file, without storing them in the class. If you make them static, the compiler cannot optimize them away (a static var must be available the whole execution of the program). Besides, constexpr can be "recreated" many times, since they won't even be there anyways at run time (most likely, unless odrused).

Reply:
hmm then it's probably better just to move them to the various operators and just use constexpr, even if it has to be defined multiple times.